services/funfuzz/recipes/set_bashrc_options.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+pushd "$HOME"
+
+cat << EOF >> .bashrc
```

Comment:
Use here: `'EOF'`

Reply:
This can now be again `EOF` without single quotes, since you moved the `PATH` stuff to the `Dockerfile`. It was preventing expanding the vars during the catenation.